setup.py

```diff
@@ -1034,8 +1031,7 @@ def detect_simple_extensions(self):
         self.add(Extension('_opcode', ['_opcode.c']))
         # asyncio speedups
         self.add(Extension("_asyncio", ["_asynciomodule.c"]))
-        # _abc speedups
-        self.add(Extension("_abc", ["_abc.c"]))
+        # _abc module is built by Modules/Setup.bootstrap
```

Comment:
Nit: I'd merge the comments regarding time, _abc, and pwd being built by Modules/Setup.bootstrap into a single comment. IMO, that information make more sense when presented in a single comment, rather than scattered around `detect_simple_extensions`.

Reply:
db3e45e1659767efc997c77c93f6c9611cfe3f94 consolidates all comments in the main detect function.